tools/ports/zlib.py

```diff
@@ -30,7 +31,7 @@ def create(final):
 
 
 def clear(ports, settings, shared):
-  shared.Cache.erase_lib('libz.a')
```

Comment:
Why this change?   I think it need to be `erase_lib` so that it knows to look inside the lib directory.